docs/taskruns.md

```diff
@@ -96,6 +96,11 @@ spec:
           - --destination=gcr.io/my-project/gohelloworld
 ```
 
+If multiple `steps` are defined in the `Task` invoked by the `TaskRun`, we will see the
+`status.steps` of the `TaskRun` displayed in the same order as they are defined in
+`spec.steps` of the `Task`, when the `TaskRun` is accessed by the `get` command, e.g.
+`kubectl get taskrun <name> -o yaml`. Replace \<name\> with the name of the `TaskRun`.
```

Comment:
thanks for adding this! could you put it in a `status` section on it's own, maybe between the `syntax` and `ordering` sections, and add a link to the new section in the overview at the top of the file? https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md#pipelines

This will give us a place to start putting more docs about the status in the future as well.

Reply:
This is a change on the file taskruns.md, not pipeline.md.
Do you actually mean to add a section in taskruns.md?

Reply:
whoops my bad! no taskruns is the right place :D